examples/nft-nd/nft.go

```diff
@@ -87,64 +93,46 @@ func BalanceOf(holder interop.Hash160) int {
 		panic("bad owner address")
 	}
 	ctx := storage.GetReadOnlyContext()
-	tokens := getTokensOf(ctx, holder)
-	return len(tokens)
+	return getBalanceOf(ctx, mkBalanceKey(holder))
 }
 
-// getTokensOf is an internal implementation of TokensOf, tokens are stored
-// as a serialized slice of strings in the DB, so it gets and unwraps them
-// (or returns an empty slice).
-func getTokensOf(ctx storage.Context, holder interop.Hash160) []string {
-	var res = []string{}
-
-	key := mkAccountKey(holder)
-	val := storage.Get(ctx, key)
+func getBalanceOf(ctx storage.Context, balanceKey []byte) int {
+	val := storage.Get(ctx, balanceKey)
 	if val != nil {
-		res = std.Deserialize(val.([]byte)).([]string)
+		return val.(int)
 	}
-	return res
+	return 0
 }
 
-// setTokensOf saves current tokens owned by account if there are any,
-// otherwise it just drops the appropriate key from the DB.
-func setTokensOf(ctx storage.Context, holder interop.Hash160, tokens []string) {
-	key := mkAccountKey(holder)
-	if len(tokens) != 0 {
-		val := std.Serialize(tokens)
-		storage.Put(ctx, key, val)
+func addToBalance(ctx storage.Context, holder interop.Hash160, amount int) {
+	key := mkBalanceKey(holder)
+	old := getBalanceOf(ctx, key)
+	old += amount
+	if old > 0 {
+		storage.Put(ctx, key, old)
 	} else {
 		storage.Delete(ctx, key)
 	}
 }
 
-// setTokens saves minted token if it is not saved yet.
-func setTokens(ctx storage.Context, newToken string) {
-	key := mkTokensKey()
-	var tokens = []string{}
-	val := storage.Get(ctx, key)
-	if val != nil {
-		tokens = std.Deserialize(val.([]byte)).([]string)
-	}
-	for i := 0; i < len(tokens); i++ {
-		if util.Equals(tokens[i], newToken) {
-			return
-		}
-	}
-	tokens = append(tokens, newToken)
-	val = std.Serialize(tokens)
-	storage.Put(ctx, key, val)
+// addToken saves current tokens owned by account if there are any,
+// otherwise it just drops the appropriate key from the DB.
+func addToken(ctx storage.Context, holder interop.Hash160, token []byte) {
+	key := mkAccountKey(holder)
```

Comment:
And it's more of `mkAccountPrefix` now with the new scheme rather than `mkAccountKey`.